transactions/src/main/resources/schema/wildfly-txn_3_1.xsd

```diff
@@ -0,0 +1,337 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For version bumps it would be much better if we could split the version bump out from the actual change being made - looking at the history of this schema it will not be possible to tell how this schema differs from the 3.0 version of the schema without manually comparing them.  

If we can split these into two commits then we have a clean version bump leading to identical models followed by the changes made to the new version of the schema / model.